src/actions/public/L1/readFinalizedWithdrawals.test.ts

```diff
@@ -0,0 +1,22 @@
+import { expect, test } from 'vitest'
+import { publicClient } from '../../../_test/utils.js'
+import { base } from '../../../chains/base.js'
+import { readFinalizedWithdrawals } from './readFinalizedWithdrawals.js'
+
+test('read finalized withdrawals', async () => {
+  const finalizedWithdrawal = await readFinalizedWithdrawals(publicClient, {
+    l2Chain: base,
+    withdrawalHash: '0xEC0AD491512F4EDC603C2DD7B9371A0B18D4889A23E74692101BA4C6DC9B5709',
+  })
+
+  expect(finalizedWithdrawal).toEqual(true)
+})
+
+test('raises error if not finalized', async () => {
```

Comment:
sorry @Sabnock01 nit can we fix this :) 
```suggestion
test('returns false if not finalized', async () => {
```